overrides/home.html

```diff
@@ -155,7 +155,7 @@ <h1>Knative Components</h1>
         Eventing work together to automate and manage tasks and applications.
       </p>
       <div class="components-content">
-        <img class="components-img" src="images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
+        <img class="components-img" src="../docs/images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
```

Comment:
```suggestion
        <img class="components-img" src="images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
```
There shouldn't be any need to change this, can you revert this change and we can see if the home page builds properly?

Reply:
yes , but this also works right , deploy preview is on point @Cali0707

Reply:
yes, but I would rather not make unnecessary changes -  can you revert this change since it shouldn't be needed?

Reply:
it is fine now @Cali0707